java/elastic-search/src/main/java/dev/sunbirdrc/elastic/ElasticServiceImpl.java

```diff
@@ -264,7 +264,7 @@ public RestStatus deleteEntity(String index, String osid) {
             readMap.put(Constants.STATUS_KEYWORD, Constants.STATUS_INACTIVE);
             response = getClient(indexL).update(new UpdateRequest(indexL, searchType, osid).doc(readMap), RequestOptions.DEFAULT);
         } catch (NullPointerException | IOException e) {
-            logger.error("exception in deleteEntity {}", e);
+            logger.error("exception in deleteEntity {}", e.getMessage());
```

Comment:
either throw the exception or log the stack trace please. Just printing the message would not help anyone.